_docs/style/date-time.md

```diff
@@ -2,65 +2,104 @@
 title: Date & Time
 layout: docs
 category: Style
-usage: |
-  Always use human-friendly formatting when displaying date and/or time.
+usage: Rackspace is a global company, so it is critical to ensure that we represent time as clearly and consistently as possible to our customers.
+  
 preview-image:
 resource: true
 ---
 
 {% include toc.html %}
 
-Always use human-friendly formatting when displaying date and/or time.
 
-# Date
+### Date
```

Comment:
Any reason to start at L3 vs L2?

Reply:
@nimbinatus yes -- it's because the page already has an H1 and H2.